Makefile

```diff
@@ -41,7 +41,8 @@ mypy:
 
 pylint:
 	$(BIN)pylint --version
-	$(BIN)pylint $(CODE)
+	$(BIN)pylint $(CODE) --disable=unspecified-encoding
```

Comment:
the pylint config is here: https://github.com/facebookincubator/submitit/blob/dac5c6b76a81401aa7d44a32357872ebf92b968a/pyproject.toml#L21

but maybe it's a valid concern ? Not everyone is using utf8 as default enconding

Reply:
it could be valid for non-test stuff (tests are most of the cases), then again... not so sure :s is it better to force utf8 or not...

Reply:
so dunno, i'd say no until someone complains

Reply:
ok, good for me.